tailwind.config.ts

```diff
@@ -0,0 +1,104 @@
+import type { Config } from 'tailwindcss';
+
+export default {
+  content: [],
+  theme: {
+    colors: {
+      green: {
+        100: '#EDF2EB',
+        200: '#C5E5B4',
+        300: '#C5E5B4',
+        400: '#84BA64',
+        500: '#5FA04E',
+        600: '#417E38',
+        700: '#2C682C',
+        800: '#2C682C',
+        900: '#1A3F1D',
+      },
+      neutral: {
+        100: '#F6F7F9',
+        200: '#E9EDF0',
+        300: '#D9E1E4',
+        400: '#CBD4D9',
+        500: '#B1BCC2',
+        600: '#929FA5',
+        700: '#6E7B83',
+        800: '#556066',
+        900: '#2C3437',
+      },
+      danger: {
+        100: '#FBF1F0',
+        200: '#FAD3D4',
+        300: '#FAB6B7',
+        400: '#FA8E8E',
+        500: '#F65354',
+        600: '#DE1A1B',
+        700: '#B80C0C',
+        800: '#900E0E',
+        900: '#661514',
+      },
+      warning: {
+        100: '#FDF3E7',
+        200: '#FAD9B0',
+        300: '#F5BC75',
+        400: '#E99C40',
+        500: '#D07912',
+        600: '#AE5F00',
+        700: '#8B4D04',
+        800: '#683D08',
+        900: '#4D2F0B',
+      },
+      info: {
+        100: '#E9F4FA',
+        200: '#BCE6FC',
+        300: '#8ED4F8',
+        400: '#52BAED',
+        500: '#229AD6',
+        600: '#0C7BB3',
+        700: '#066291',
+        800: '#074D71',
+        900: '#0A3953',
+      },
+      accent1: {
+        100: '#F7F1FB',
+        200: '#EAD9FB',
+        300: '#DBBDF9',
+        400: '#C79BF2',
+        500: '#AF74E8',
+        600: '#9756D6',
+        700: '#7D3CBE',
+        800: '#642B9E',
+        900: '#361B52',
+      },
+      accent2: {
+        100: '#FBF0F4',
+        200: '#FBD4E6',
+        300: '#FBB4D2',
+        400: '#F68BB7',
+        500: '#ED5393',
+        600: '#D6246E',
+        700: '#B01356',
+        800: '#8B1245',
+        900: '#411526',
+      },
+      white: '#FFFFFF',
+      black: '#0D121C',
+    },
+    fontSize: {
```

Comment:
Do we leave it as a single quote or just like that?

Reply:
Wdym?

Reply:
code before (doesn't display on github) have font size write like that `XS: '10rem'` and `'3XS': '30rem'`. So should I put all fontSize in single quote ?

Reply:
No. If you put on single quote, prettier will automatically remove them for you. Single Quote in object property names is only used for composite property names (that have special characters, spaces or symbols).

Reply:
Ok perfect